hudi-spark-datasource/hudi-spark/src/main/java/org/apache/hudi/QuickstartUtils.java

```diff
@@ -190,6 +192,30 @@ public List<HoodieRecord> generateUpdates(Integer n) {
       }).collect(Collectors.toList());
     }
 
+    /**
+     * Generates new updates, one for each of the keys above
+     * list
+     *
+     * @param n Number of updates (must be no more than number of existing keys)
+     * @return list of hoodie record updates
+     */
+    public List<HoodieRecord> generateUniqueUpdatesForExample(Integer n) {
```

Comment:
what "example" matters? why can't we make it generic to be used even in our tests?

Reply:
oh, this datagenerator is used just for QuickStartUtils. so, you can just name this "generateUniqueUpdates". would suffice.